deps/build.jl

```diff
@@ -1,8 +1,17 @@
-if Sys.KERNEL === :Darwin && (!success(`command -v xcode-select`) || isempty(readchomp(`xcode-select -p`)))
+using Compat
+using Compat.Sys: isapple, iswindows
+
+if isapple() && (!success(`command -v xcode-select`) || isempty(readchomp(`xcode-select -p`)))
     error("Building Rmath on macOS requires the Xcode command line tools to be installed.\n",
           "You can install them from the command line using `xcode-select --install`.")
 end
 
+if !iswindows() && !(success(`command -v make`) || success(`command -v gmake`))
+    error("Building Rmath requires GNU Make to be installed. You can install it from your\n",
```

Comment:
I don't think `command` is available on non-BSD.

Reply:
could just try:
```
success(`make -v`)
```

Reply:
`command` is POSIX, but I guess it needs to be ``success(`sh -c "command -v make"`)`` to be fully correct. Won't ``success(`$(BinDeps.MAKE_CMD) -v`)`` fail with `ENOENT` if Make isn't installed?